packages/host/app/lib/stack-item.ts

```diff
@@ -149,3 +149,8 @@ export class StackItem {
     });
   }
 }
+
+export function isIndexCard(stackItem: StackItem) {
+  let realmURL = stackItem.card[stackItem.api.realmURL];
+  return stackItem.card.id === `${realmURL!.href}index`;
```

Comment:
let's not assume realmURL exists. in general `!` usage is a red flag.

this would be better:
```suggestion
if (!realmURL) {
  return false;
}
return stackItem.card.id === `${realmURL.href}index`;
```